tests/example_listings.py

```diff
@@ -7,31 +12,28 @@
 settings.load_profile("unit")
 
 
+@composite
+def requirements_list(draw, length, answers=False):
+    if answers:
+        elements = booleans() if length is not None else one_of(booleans(), none())
+    else:
+        elements = text(min_size=1, alphabet='abcdefgh', average_size=10)
```

Comment:
We want to exclude spaces since we drop space-only strings when stripping whitespace. And using a smaller alphabet also seems to speed up the initialisation (I'm not sure if it might be explained by something else, but I've seen a +1s difference when running without alphabet and average_size. I suspect it's caused by sampling unicode characters)